src/tools/execution-results.h

```diff
@@ -99,7 +99,7 @@ struct ExecutionResults {
     optimizedResults.get(wasm);
     if (optimizedResults != *this) {
       std::cout << "[fuzz-exec] optimization passes changed execution results";
-      abort();
+      exit(1);
```

Comment:
Yeah sorry.. I think you should split out the abort -> exit change since it logically something we want to do separately to wasm64.